src/main/java/uk/gov/companieshouse/exception/DissolutionNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.exception;
+
+public class DissolutionNotFoundException extends RuntimeException {
```

Comment:
What's your thoughts on making this a checked Exception rather than runtime to ensure that it is defended against?